test/openAPI/features/wallet_consent.feature

```diff
@@ -0,0 +1,33 @@
+@method=POST @endpoint=/linked-authorization/consent
+Feature: The endpoint to send the accepted user claims and permitted scopes.
+
+  @smoke @unit @positive
+  Scenario: Successfully sends the accepted user claims and permitted scopes smoke type test
+    Given Wants to send the accepted user claims and permitted scopes
+    And The linkedTransactionId is generated before POST /linked-authorization/consent
```

Comment:
/linked-authorization/authenticate should also be performed for the linkedTransactionId before using it in /linked-authorization/consent endpoint call

Reply:
Changes added in the [PR](https://github.com/GovStackWorkingGroup/bb-identity/pull/46)